packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2306,24 +2306,39 @@ export const handlers = [
   }),
   http.get('*/monitor/services', () => {
     const response = {
-      data: [{ service_type: 'linode' }],
+      data: [
+        {
+          label: 'Linode',
+          service_type: 'linode',
+        },
+        {
+          label: 'Databases',
+          service_type: 'dbaas',
+        },
+      ],
     };
 
     return HttpResponse.json(response);
   }),
-  http.get('*/monitor/services/:serviceType/dashboards', () => {
+  http.get('*/monitor/services/:serviceType/dashboards', ({ params }) => {
     const response = {
-      data: [
+      data: [] as any[],
+    };
+    if (params.serviceType == 'linode') {
+      response.data.push(
         dashboardFactory.build({
           label: 'Linode Dashboard',
           service_type: 'linode',
-        }),
+        })
+      );
+    } else if (params.serviceType == 'dbaas') {
```

Comment:
```suggestion
    } else if (params.serviceType === 'dbaas') {
```

Reply:
@ankitaakamai   Please refer our standard checklist for some of these best practices cases. "https://collaborate.akamai.com/confluence/pages/viewpage.action?spaceKey=DI&title=ACLP+PR+standard+checklists"

Reply:
Updated